deps.go

```diff
@@ -14,6 +15,7 @@ import (
 	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
 	_ "github.com/jewzaam/go-cosmosdb/cmd/gencosmosdb"
 	_ "github.com/jstemmer/go-junit-report"
+	_ "github.com/onsi/ginkgo/v2/ginkgo"
```

Comment:
Wondering why did the authors of ginkgo decide to have this `/v2` prefix instead of releasing new major version?